src/main/java/org/terasology/metalrenegades/population/system/PopulationSystem.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.metalrenegades.population.system;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.dynamicCities.buildings.components.SettlementRefComponent;
+import org.terasology.dynamicCities.settlements.events.SettlementRegisterEvent;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.metalrenegades.ai.component.FactionAlignmentComponent;
+import org.terasology.metalrenegades.ai.component.HomeComponent;
+import org.terasology.metalrenegades.ai.event.CitizenSpawnedEvent;
+import org.terasology.metalrenegades.population.component.MetalRenegadesPopulationComponent;
+
+@RegisterSystem(value = RegisterMode.AUTHORITY)
+public class PopulationSystem extends BaseComponentSystem {
+
+
+    Logger logger = LoggerFactory.getLogger(PopulationSystem.class);
+
+    @ReceiveEvent
+    public void onSettlementRegisterEvent(SettlementRegisterEvent buildingEntitySpawnedEvent, EntityRef entityRef) {
+        entityRef.addComponent(new MetalRenegadesPopulationComponent());
+    }
+
+    @ReceiveEvent
+    public void citizenSpawned(CitizenSpawnedEvent event, EntityRef citizen,
+                               FactionAlignmentComponent factionAlignmentComponent, HomeComponent homeComponent) {
+
+        EntityRef homeEntity = homeComponent.building;
+        SettlementRefComponent settlementRefComponent = homeEntity.getComponent(SettlementRefComponent.class);
+        EntityRef settlementEntity = settlementRefComponent.settlement;
```

Comment:
It might be cleaner to pass the settlement entity and the faction alignment through the `CitizenSpawnedEvent`. This event is only emitted [here](https://github.com/Terasology/MetalRenegades/blob/ff1b0586c0434d411ca4d8d928668cca8e0d079d/src/main/java/org/terasology/metalrenegades/ai/system/CitizenSpawnSystem.java#L142) and all that information is available.